dipy/io/dpy.py

```diff
@@ -113,34 +134,54 @@ def write_tracks(self, tracks):
             self.offsets[-tracks._offsets.shape[0] - 1] + \
             tracks._offsets + tracks._lengths
 
-    def read_track(self):
+    def read_track(self) -> np.array:
         """ read one track each time
+        Args:
+                None
+        Returns:
+                a numpy array with shape (N, 3)
         """
         off0, off1 = self.offsets[self.offs_pos:self.offs_pos + 2]
         self.offs_pos += 1
         return self.tracks[off0:off1]
 
-    def read_tracksi(self, indices):
+    def read_tracksi(self, indices: List[int]) -> Streamlines:
         """ read tracks with specific indices
+        Args:
+                indices: a list of indices
+        Returns:
+                a Streamlines object
         """
         tracks = Streamlines()
         for i in indices:
             off0, off1 = self.offsets[i:i + 2]
             tracks.append(self.tracks[off0:off1])
         return tracks
 
-    def read_tracks(self):
+    def read_tracks(self) -> Streamlines:
         """ read the entire tractography
+        Args:
```

Comment:
Can you follow numpy documentation standard : https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Okay